src/crimson/osd/ops_executer.h

```diff
@@ -127,7 +133,13 @@ class OpsExecuter : public seastar::enable_lw_shared_from_this<OpsExecuter> {
       return pimpl->get_map_epoch();
     }
     entity_inst_t get_orig_source_inst() const final {
-      return pimpl->get_orig_source_inst();
```

Comment:
So `get_orig_source_inst()` is useless in crimson, right?

How about killing it, even by `#ifdef WITH_SEASTAR` and adding the comment there?

Reply:
#51318

Reply:
Ah, I was missing that `get_orig_source_inst()` is trying to access the message's connection.